start.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+# Function to generate a random string
+random_string() {
+    openssl rand -base64 24 | tr -dc 'A-Za-z0-9' | head -c 32
+}
+
+# Ensure compose files exist
+ensure_compose() {
+  if [ -z "$BRANCH" ]; then
+    BRANCH="feat/simplified-install-script"
+  fi
+  [ ! -f ./docker-compose.yaml ] && wget "https://raw.githubusercontent.com/dyrector-io/dyrectorio/refs/heads/$BRANCH/docker-compose.yaml"
+  [ "$DEPLOY_TEST_MAIL" == "y" ] && [ ! -f ./distrib/compose/docker-compose.mail.yaml ] && mkdir -p ./distrib/compose && wget "https://raw.githubusercontent.com/dyrector-io/dyrectorio/refs/heads/$BRANCH/docker-compose.mail.yaml" -O ./distrib/compose/docker-compose.mail.yaml
+  [ "$DEPLOY_TRAEFIK" == "y" ] && [ ! -f ./distrib/compose/docker-compose.traefik.yaml ] && mkdir -p ./distrib/compose && wget "https://raw.githubusercontent.com/dyrector-io/dyrectorio/refs/heads/$BRANCH/distrib/compose/docker-compose.traefik.yaml" -O ./distrib/compose/docker-compose.traefik.yaml
+  [ "$ADD_TRAEFIK_LABELS" == "y" ] && [ ! -f ./distrib/compose/docker-compose.traefik-labels.yaml ] && mkdir -p ./distrib/compose && wget "https://raw.githubusercontent.com/dyrector-io/dyrectorio/refs/heads/$BRANCH/distrib/compose/docker-compose.traefik-labels.yaml" -O ./distrib/compose/docker-compose.traefik-labels.yaml
+}
+
+# Check if running in an interactive terminal
+if [ -t 0 ]; then
```

Comment:
Missing the user guide; how to run in interactive mode and missing the flags help page too or something which can tell a user how to use this script in general.

Reply:
Interactive mode is the default if you are in a regular TTY, if this is ran in a container or a non-standard shell and the user doesn't provide parameters if gives info how this could be still executed. Otherwise the point of a wizard is that you **don't** have to know anything beforehand and it guides you through the setup process, but there is a help few lines down here, if you mean to extend that one.